app/uk/gov/hmrc/economiccrimelevyregistration/models/Registration.scala

```diff
@@ -23,11 +23,22 @@ final case class Registration(
   internalId: String,
   entityType: Option[EntityType],
   meetsRevenueThreshold: Option[Boolean],
+  amlSupervisor: Option[AmlSupervisor],
   incorporatedEntityJourneyData: Option[IncorporatedEntityJourneyData],
   soleTraderEntityJourneyData: Option[SoleTraderEntityJourneyData],
   partnershipEntityJourneyData: Option[PartnershipEntityJourneyData]
 )
 
 object Registration {
   implicit val format: OFormat[Registration] = Json.format[Registration]
+
+  def empty(internalId: String): Registration = Registration(
+    internalId = internalId,
+    entityType = None,
+    meetsRevenueThreshold = None,
+    amlSupervisor = None,
```

Comment:
reminder to update the backend model 😄

Reply:
Updated Registration model on backend microservice - see separate PR